src/frontends/pytorch/src/op/shape_as_tensor.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_shape_as_tensor(const NodeContext& context) {
+    num_inputs_check(context, 1, 1);
+    auto input = context.get_input(0);
+    auto shape = context.mark_node(std::make_shared<opset10::ShapeOf>(input));
```

Comment:
please double check what output type should be. And I would suggest explicitly set it here

Reply:
Good point, agree. In tests we treat i64 and i32 as equal, but for this operation it is important to have same type as pytorch

Reply:
Done! It should be i64 (default of ShapeOf) but for the sake of readability I explicitly added it to the function call.